src/rate_limiter.cc

```diff
@@ -302,6 +314,7 @@ RateLimiter::InitializePayloadQueues(const TritonModelInstance* instance)
   } else {
     max_queue_delay_microseconds = 0;
   }
+  PayloadQueue* payload_queue;
```

Comment:
You may want to initialize this immediately to a nullptr to avoid unexpected behavior (e.g. segfault) if the code changes later.

Reply:
Update commit: [Init rate limiter ptr to null once declared](https://github.com/triton-inference-server/core/pull/165/commits/15781d6d2a08c7053bb9fa0b0ec304bb3de7fb6e)